CuratedContent/TuringWay.md

```diff
@@ -0,0 +1,34 @@
+# Reproducible and ethical science. 
+<!--deck text start-->
+An on line handbook to reproducible, ethical, and collaborative computational and data science.
+<!--deck text end-->
+
+#### Contributed by [Jed Brown](https://github.com/jedbrown), [Mark C. Miller](https://github.com/markcmiller86 "Mark C. Miller GitHub Profile")
+#### Publication date: June 22, 2023
+
+Resource information | Details
+:--- | :--- 
+Article title  | [The Turing Way](https://the-turing-way.netlify.app/index.html)
+Authors | The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. 
+Length | Multi-Chapter, on line handbook
+Focus | Reproducible, ethical and collaborative data science
+
+This is a wonderful handbook to reproducible, ethical, and collaborative computational and data science.
```

Comment:
"wonderful" seems a little strong.  Perhaps more techy words like "impressive" or "solid"?

Reply:
Wonderful was @jedbrown actual words. I didn't want to adjust it.